test/tree/ast/typescript/typeScriptFileParserProjectTest.ts

```diff
@@ -46,7 +46,7 @@ describe("TypeScriptFileParser real project parsing: TypeScript", () => {
                 assert(values.length > 2);
                 done();
             }).catch(done);
```

Comment:
tests should `catch(done, done)` so that failures don't have to timeout, and messages are happer